stdlib/universe/derivative_test.go

```diff
@@ -26,17 +28,6 @@ func TestDerivativeOperation_Marshaling(t *testing.T) {
 	querytest.OperationMarshalingTestHelper(t, data, op)
 }
 
-func TestDerivative_PassThrough(t *testing.T) {
-	executetest.TransformationPassThroughTestHelper(t, func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation {
-		s := universe.NewDerivativeTransformation(
-			d,
-			c,
-			&universe.DerivativeProcedureSpec{},
-		)
-		return s
-	})
-}
```

Comment:
I wasn't really sure what this test did in all honesty. We are also only testing the legacy derivative in this file because the feature flag isn't enabled in this context. For the end to end tests, it is using the new derivative. I'm not sure how to test both the new and old version for end to end tests.